modal/modal.css

```diff
@@ -0,0 +1,77 @@
+@import url("/style.css");
+@import url("/button/button.css");
+
+/* main {
+  padding: 2rem;
+  border: 1px solid black;
+} */
+
+.ct-modal {
+  padding: 0 1rem;
+  border: 1px solid var(--gray-500);
+  max-width: 350px;
+  max-height: 100%;
+  cursor: pointer;
+  border-radius: 5px;
+}
+
+.ct-modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 1rem 0;
+  border-bottom: 1px solid var(--gray-300);
+}
+
+.ct-modal-body {
+  padding: 1rem 0;
+  border-bottom: 1px solid var(--gray-300);
+}
+
+.ct-modal-footer {
+  width: 100%;
+  padding: 0.75rem 0;
+  display: flex;
+  justify-content: end;
+  flex-wrap: wrap;
+}
+.ct-modal-footer button {
+  margin: 0.25rem;
+}
+
+.ct-modal-action {
+  position: absolute;
+  width: 400px;
+  top: -100px;
```

Comment:
I think you should follow single units either go with `Px` or `rem`.

Reply:
yup changed.